events/metrics.go

```diff
@@ -0,0 +1,36 @@
+package events
+
+import (
+	pmetrics "github.com/awslabs/operatorpkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	MetricLabelGroup       = "group"
+	MetricLabelKind        = "kind"
+	MetricLabelEventType   = "type"
+	MetricLabelEventReason = "reason"
+)
+
+const (
+	MetricNamespace = "operator"
+	MetricSubsystem = "event"
+)
+
+// Cardinality is limited to # objects * # conditions
+var EventCount = pmetrics.NewPrometheusCounter(
+	metrics.Registry,
+	prometheus.CounterOpts{
+		Namespace: MetricNamespace,
+		Subsystem: MetricSubsystem,
+		Name:      "count",
```

Comment:
The correct Prometheus name for a counter always ends with `total`, so something like `operator_node_event_count`